### DISCUSSION TITLE
Can I use oap 9.1 with java agent 8.10 together?

### DISCUSSION THREAD

**Discussion by a1242902560** [apache/skywalking#9325] [Q&A] [answered]
I want to know the relationship between oap-server and  java agent, why after that, their version 
 relationship does not correspond

**Comment by lujiajing1126** [comment, marked as answer]
> I want to know the relationship between oap-server and java agent, why after that, their version relationship does not correspond

See compatibility matrix here

https://skywalking.apache.org/docs/main/latest/en/setup/service-agent/agent-compatibility/

**Reply by a1242902560** [reply]
Thank you.  Blame me for not looking at the documentation
